lookout/style/typos/cmdline_tools.py

```diff
@@ -0,0 +1,180 @@
+    Add config argument for data preparation.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "--preparation-config", required=False, type=json.loads, default="{}",
+        help="Config for data preparation in json format.")
+
+
+def add_fasttext_config_arg(my_parser: ArgumentParser) -> None:
+    """
+    Add config argument for fasttext training.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "--fasttext-config", required=False, type=json.loads, default="{}",
+        help="Config for fasttext training in json format.")
+
+
+def add_datasets_config_arg(my_parser: ArgumentParser) -> None:
+    """
+    Add config argument for train and test datasets generation.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "--datasets-config", required=False, type=json.loads, default="{}",
+        help="Config for datasets generation in json format.")
+
+
+def add_data_path_arg(my_parser: ArgumentParser) -> None:
+    """
+    Add data path argument.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "--data-path", required=True, type=str,
+        help=".csv dump of a Dataframe with columns Columns.Split and Columns.Frequency.")
+
+
+def add_save_model_path_arg(my_parser: ArgumentParser) -> None:
+    """
+    Add save model path argument.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "--save-model-path", required=True, type=str,
+        help="Path to save the trained model to (.asdf).")
+
+
+def cli_train_fasttext(data_path: str, fasttext_config: Mapping[str, Any]) -> None:
+    """Entry point for `train_fasttext`."""
+    train_fasttext(pandas.read_csv(data_path, index_col=0, keep_default_na=False),
+                   fasttext_config)
+
+
+def cli_get_datasets(data_path: str, datasets_config: Mapping[str, Any]) -> None:
+    """Entry point for `get_datasets`."""
+    get_datasets(pandas.read_csv(data_path, index_col=0, keep_default_na=False), datasets_config)
+
+
+def cli_train_corrector(data_path: str, test_data_path: str, vocabulary_path: str,
+                        frequencies_path: str, fasttext_path: str, save_model_path: str) -> None:
+    train = pandas.read_csv(data_path, index_col=0, keep_default_na=False)
+    test = pandas.read_csv(test_data_path, index_col=0, keep_default_na=False)
+    model = train_and_evaluate(train, test, vocabulary_path, frequencies_path, fasttext_path)
+    model.save(save_model_path, series=0.0)
+
+
+def create_parser() -> ArgumentParser:
+    """
+    Create a parser for the lookout.style.typos utility.
+
+    :return: an ArgumentParser with an handler defined in the handler attribute.
+    """
+    parser = ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatterNoNone)
+
+    # General options
+    slogging.add_logging_args(parser)
```

Comment:
Can you please add "erase_args" boolean parameter to `add_logging_args` in sdk-ml so that it delattr-s those three arguments automatically (separately of this PR).